---
title: "Raw Sequence QC"
permalink: /sequenceQC/
toc: true
toc_label: "Sequence Quality Control"
toc_sticky: true
---


Below are a list of tools for performing quality control of sequence data. All of these tools can be downloaded using [conda](https://docs.conda.io/en/latest/). 

### Sequence Data Quality Analysis Tools

* [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) is a tool for assessing raw sequence quality.
* [Kat](https://www.earlham.ac.uk/kat-tools) is a suite of tools for generating, analyzing, and comparing k-mer spectra produced from sequence data. Spectra includes determing sequence completeness, sequence bias, identifying contaminants, validating assemblies. 
* [Jellyfish](https://www.cbcb.umd.edu/software/jellyfish/) is a tool for counting k-mers in sequence data
* [GenomeScope](http://qb.cshl.edu/genomescope/) is both a R package and website which estimates genome heterozygosity, repeat content, and size using kmer based statistical approaches
* [kmergenie](http://kmergenie.bx.psu.edu/) is a tool for estimating the best k-mer length for de novo genome assembly

### Sequence Data Filtering Tools

* [TrimAL](http://trimal.cgenomics.org/)  
* [Trimmomatic](http://www.usadellab.org/cms/?page=trimmomatic)  
* [AfterQC](https://github.com/OpenGene/AfterQC)  
* [AdapterRemoval](https://github.com/MikkelSchubert/adapterremoval)  
* [Trim Galore!](https://github.com/FelixKrueger/TrimGalore)  
* [BBDuk](https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
* [QC3](https://github.com/slzhao/QC3)


